app/helpers/latex_helper.rb

```diff
@@ -0,0 +1,7 @@
+module LatexHelper
+  def generate_pdf(template:)
+    raise "LATEX_CONTAINER_NAME is not set" if ENV['LATEX_CONTAINER_NAME'].nil?
```

Comment:
Use ' around text - " if the text contains string interpolation or escape characters. Using ' will be faster where these do not exist.